docs/user/collision-shape-for-physics.rst

```diff
@@ -51,8 +43,40 @@ Posting the query and polling for its result are done through the same function.
 Finally due to the above properties, the number of query points posted from a particular owner should be kept consistent across frames.
 The helper classes always submit a fixed number of points this frame, so satisfy this criteria.
 
-Gerstner Waves CPU
-------------------
+
+Compute Shape Queries (GPU)
+---------------------------
+
+This is the default and recommended choice for when a GPU is present.
+Query positions are uploaded to a compute shader which then samples the ocean data and returns the
+desired results.
+The result of the query accurately tracks the height of the surface, including all wave components and depth caches and other Crest features.
+
+
+Baked FFT Data (CPU)
+--------------------
+
```

Comment:
If this is going to be a preview feature, then there an admonition for it:
```rst
.. admonition:: Preview
```

Reply:
yeah i guess it might be a good shout as the baked data might get breaking changes after more testing?

would you have a suggestion how to use that tag - should the entire section be indented, or should i add a new line inside the section saying that its in preview and use the tag there?

Reply:
Sure. Content needs to be indented and there needs to be an empty line between tag and content. It is used on the ocean simulation page. Example:
```rst
.. admonition:: Preview

   Some content.
```

Reply:
Ok so I'll indent the body and put it on the tag, but not the header.

Reply:
done now

Reply:
I've changed it so preview contains just a short note: This feature is in preview and may change in the future.